test/Microsoft.Health.Fhir.Shared.Tests.E2E/Rest/AnonymizedExportUsingAcrTests.cs

```diff
@@ -0,0 +1,479 @@
+// -------------------------------------------------------------------------------------------------
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License (MIT). See LICENSE in the repo root for license information.
+// -------------------------------------------------------------------------------------------------
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Net;
+using System.Net.Http;
+using System.Net.Http.Headers;
+using System.Security.Cryptography;
+using System.Text;
+using System.Threading;
+using System.Threading.Tasks;
+using Hl7.Fhir.Model;
+using Hl7.Fhir.Serialization;
+using Microsoft.Azure.ContainerRegistry;
+using Microsoft.Azure.ContainerRegistry.Models;
+using Microsoft.Azure.Storage;
+using Microsoft.Azure.Storage.Auth;
+using Microsoft.Azure.Storage.Blob;
+using Microsoft.Health.Fhir.Core.Extensions;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export;
+using Microsoft.Health.Fhir.Core.Features.Operations.Export.Models;
+using Microsoft.Health.Fhir.Core.Models;
+using Microsoft.Health.Fhir.TemplateManagement.Utilities;
+using Microsoft.Health.Fhir.Tests.Common;
+using Microsoft.Health.Fhir.Tests.Common.FixtureParameters;
+using Microsoft.Health.Fhir.Tests.E2E.Common;
+using Microsoft.Health.Fhir.Tests.E2E.Rest.Metric;
+using Microsoft.Health.Test.Utilities;
+using Microsoft.Rest;
+using Newtonsoft.Json;
+using Xunit;
+using FhirGroup = Hl7.Fhir.Model.Group;
+using Task = System.Threading.Tasks.Task;
+
+namespace Microsoft.Health.Fhir.Tests.E2E.Rest
+{
+    [Trait(Traits.Category, Categories.AnonymizedExport)]
+    [HttpIntegrationFixtureArgumentSets(DataStore.All, Format.Json)]
+    public class AnonymizedExportUsingAcrTests : IClassFixture<ExportTestFixture>
+    {
+        private const string TestRepositoryName = "anonymizationconfigs";
```

Comment:
Can we get the repository name from an environment variable instead of hardcoding it into the test file? We shouldn't be linking our code to specific Azure resources.